solr/server/etc/jetty-http.xml

```diff
@@ -40,9 +40,9 @@
         </Arg>
         <Set name="host"><Property name="solr.host.bind" default="127.0.0.1"/></Set>
         <Set name="port"><Property name="solr.port.listen" default="8983" /></Set>
-        <Set name="idleTimeout"><Property name="solr.jetty.http.idleTimeout" default="120000"/></Set>
-        <Set name="acceptorPriorityDelta"><Property name="solr.jetty.http.acceptorPriorityDelta" default="0"/></Set>
-        <Set name="acceptQueueSize"><Property name="solr.jetty.http.acceptQueueSize" default="0"/></Set>
+        <Set name="idleTimeout"><Property name="solr.jetty.http.timeout.ms" default="120000"/></Set>
```

Comment:
I'm not sure I like removing the "idle", it makes the parameter confusing, as the user wouldn't know what kind of timeout this is (my first thought would be this is a request timeout)